### DISCUSSION TITLE
Should we pass the Library Title or Library Name inside this PnP Function " Get-PnPListItem -List"

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [pnp/powershell#2796] [Q&A]
Should we pass the Library/List Title or Library/List Name inside this PnP Function

[Get-PnPListItem -List](url)

now based on my test both will work, so could the pnp fail safe to get the library title incase the library name is missing?

Thanks

**Comment by AndersRask** [comment]
Actually the best thing (for performance reasons) is to pass it a list object 

$list = Get-PnPList -Identity <listpipebind>
$listItems = Get-PnPListItem -List $list

list pipebind can be guid, name or url of the list
